### PR TITLE
Move security plugin from Plugin to Management section on left navigation menu

### PR DESCRIPTION
**Pull request by Hailong-am**
### Description
Move security plugin from Plugin to Management section on left navigation menu

### Category
Enhancement

### Why these changes are required?
It is part of [Navigation changes for administrative features](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4133)

### What is the old behavior before changes and new behavior after changes?

**Before**
image

**After**
image

### Issues Resolved
https://github.com/opensearch-project/security-dashboards-plugin/issues/1435

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/plugin.ts [hunk 1]
```diff
@@ -112,12 +114,20 @@ export class SecurityPlugin
...
       });
+
+      if (deps.managementOverview) {
+        deps.managementOverview.register({
+          id: PLUGIN_NAME,
+          title: 'Security',
+          order: 9050,
+          description: i18n.translate('security.securityDescription', {
+            defaultMessage:
+              'Configure how users access data in OpenSearch with authentication, access control and audit logging.',
```
Comment (Hailong-am) [inline public/plugin.ts:127]: wording is still under review.

**Review by cwperks — COMMENTED** [review]
@Hailong-am This change looks good to me. I will re-run the failing checks as soon as new artifacts become available so that they do not get stuck on this error:

> java.lang.NoClassDefFoundError: org/opensearch/core/common/io/stream/BaseStreamInput

**Comment by cwperks** [comment]
@Hailong-am Should this PR be backported to 2.x? Is this targeted for 2.9?

**Review by cwperks — APPROVED** [review]

**Comment by Hailong-am** [comment]
> @Hailong-am Should this PR be backported to 2.x? Is this targeted for 2.9?

@cwperks thanks for reviewing the code. It's target for 2.9, please help to add backport 2.x label.

**Comment by cwperks** [comment]
Added the backport 2.x label. @RyanL1997 @cliu123 Could this PR get another review?

**Review by davidlago — APPROVED** [review]

**Pull request merged**